files/KoreBuild/scripts/KoreBuild.psm1

```diff
@@ -68,7 +68,8 @@ function Invoke-RepositoryBuild(
         $sdkVersion = __get_dotnet_sdk_version
         if ($sdkVersion -ne 'latest') {
             "{ `"sdk`": { `"version`": `"$sdkVersion`" } }" | Out-File (Join-Path $Path 'global.json') -Encoding ascii
-        } else {
+        }
```

Comment:
Do you have an extention for VSCode that formats PS files for you or do you just notice these yourself?

Reply:
VS Code formats this for me. https://marketplace.visualstudio.com/items?itemName=ms-vscode.PowerShell